integrated_channels/integrated_channel/models.py

```diff
@@ -282,6 +282,19 @@ def get_last_sync(self, include_only_errors):
         else:
             return None
 
+    @property
+    def is_valid(self):
```

Comment:
this should be a commonly defined method that is overridden. opportunities to clean this up but per our discussion in slack, too complicated to clean up now.